util/marketplace.py

```diff
@@ -160,11 +169,12 @@ def create_entitlement(self, customerId, sku):
 
     def find_stripe_subscription(self, account_number):
         """
-        Returns the stripe plan for a given account number
+        Returns the stripe plan/s for a given account number
         """
+        stripe_plans = []
         for sku in RH_SKUS:
             user_subscription = self.lookup_subscription(account_number, sku)
             if user_subscription is not None:
-                return get_plan_using_rh_sku(sku)
+                stripe_plans.append(get_plan_using_rh_sku(sku))
```

Comment:
Do these plans include RH subscriptions? Because a user should only ever have a single _Stripe_ plan.